rust/sbp2json/src/bin/sbp2json.rs

```diff
@@ -32,10 +32,10 @@ pub struct Options {
     /// Buffer output before flushing
     #[clap(short, long)]
     buffered: bool,
-
-    /// Stop on first error encountered
-    #[clap(long)]
-    fatal_errors: bool,
+    ///
+    /// what to do if encounter an error while parsing.
```

Comment:
This should follow the convention of the other docstrings, also I'm not sure how the leading whitespace will show up when this is rendered as a help string for the command line options:

```suggestion
    /// What to do if encounter an error while parsing
```

Reply:
nice catch.